bond_test.go

```diff
@@ -51,10 +63,23 @@ type User struct {
 	Username  string `db:"username"`
 }
 
+func (u User) AfterCreate(tx bond.Session) {
```

Comment:
Good point, "sess" is more transparent than "ctx" or "tx" and it does not
require a confusing explanation about problems with different contexts.

On Tue, Aug 25, 2015 at 9:17 PM, Peter Kieltyka <notifications@github.com>
wrote:

> In bond_test.go
> <https://github.com/upper/bond/pull/7#discussion_r37940945>:
>
> > @@ -51,10 +63,23 @@ type User struct {
> >  	Username  string `db:"username"`
> >  }
> >
> > +func (u User) AfterCreate(tx bond.Session) {
>
> however, I would call it.. AfterCreate(sess bond.Session) ... since the
> session can be a normal conn or a tx .. its up to the previous caller
> context
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/upper/bond/pull/7/files#r37940945>.
>